packages/client/src/app/services/content/news.service.ts

```diff
@@ -0,0 +1,47 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { MatSnackBar } from '@angular/material/snack-bar';
+import { throwError } from 'rxjs';
+import { map, catchError } from 'rxjs/operators';
+
+import { PaginateResult } from '@pulp-fiction/models/util';
+import { NewsContentModel } from '@pulp-fiction/models/content';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class NewsService {
+  private url = `/api/content/news`;
+
+  constructor(private http: HttpClient, private snackBar: MatSnackBar) { }
+
+  /**
+   * Fetches a page of news results.
+   * 
+   * @param pageNum The current page
+   */
+  public getNewsFeed(pageNum: number) {
+    return this.http.get<PaginateResult<NewsContentModel>>(`${this.url}/news-feed/${pageNum}`, {observe: 'response', withCredentials: true})
+      .pipe(map(res => {
+        return res.body;
+      }), catchError(err => {
+        this.snackBar.open(`${err.error.message}`);
```

Comment:
A thought: I see you're using the material SnackBar now instead of the AlertsService. Should we just wrap the MatSnackBar in the AlertsService?